packages/reports/addon/components/common-actions/schedule.hbs

```diff
@@ -92,122 +92,124 @@
                     </DenaliButton>
                   </div>
                 {{else}}
-                  <DenaliInputGroup @isStacked={{true}} @label="Schedule Name" class="m-b-30">
-                    <DenaliInput 
-                      @wrapperClass="w-full" 
-                      type="text" 
-                      placeholder="Schedule Name (Optional)"
-                      class="schedule__modal-name"
-                      value={{unless this.currentDisplayedRule.name this.currentDisplayedRule.name this.defaultName}}
```

Comment:
Can we do this approach? A computed property could be used as well. Also, do we have tests around the bug we had before?

Reply:
Yeah, you could have a computed property with a getter and setting backed by the persisted `name` attribute